File: js/task2/S2_ARR_05.js
function stats(nums) {
  if (nums.length === 0) return null;

  return {
      min: Math.min(...nums),
      max: Math.max(...nums),
      avg: nums.reduce((sum, num) => sum + num, 0) / nums.length
  };
}

console.log(stats([1, 2, 3, 4, 5]));    
console.log(stats([10, -5, 0, 100]));    
console.log(stats([7]));                
console.log(stats([]));                 
console.log(stats([1.5, 2.5, 3.5]));

File: python/task2/S2_ARR_05.py
def stats(nums):
  if len(nums) == 0:
      return None

  return {
      "min": min(nums),
      "max": max(nums),
      "avg": sum(nums) / len(nums)
  }

print(stats([1, 2, 3, 4, 5]))
print(stats([10, -5, 0, 100]))
print(stats([7]))
print(stats([]))
print(stats([1.5, 2.5, 3.5]))
